packages/toast/src/Toast.tsx

```diff
@@ -79,153 +79,147 @@ const variantStyles: Record<
 > = {
   [Variant.Success]: {
     toast: css`
-      background-color: ${uiColors.green.light3};
+      background-color: ${palette.green.light3};
+      border: 1px solid ${palette.green.light2};
```

Comment:
Small detail: Might be simpler to just toggle `border-color`, and leave `1px solid` in the base styles